data/tutorials/rs_00_guidelines.md

```diff
@@ -411,8 +415,8 @@ match List.map succ (l1 @ l2) with
 ```
 
 
-* Global definition with de-structuring lets should be rewritten with
- explicit pattern matching and tuples:
+* Global definition with de-structuring `lets` should be rewritten with
+ explicit pattern-matching and tuples:
```

Comment:
Is there a way to write this without using the plural for _let_?

* On one hand, `lets` is not a reserved word of the OCaml syntax. Therefore it should not be written as code.
* On the other hand, plurality is there, it has to show up but _`let`s_ seems a very bad idea.

Maybe something like this?

> Global definition with de-structuring `let` statements should be rewritten ...